server/@types/dto/index.d.ts

```diff
@@ -98,6 +98,90 @@ declare module 'dto' {
     interests: Array<FutureWorkInterestDto>
   }
 
+  export interface CreateInductionDto {
+    prisonId: string
+    workOnRelease: CreateWorkOnReleaseDto
+    previousQualifications?: CreatePreviousQualificationsDto
+    previousTraining?: CreatePreviousTrainingDto
+    previousWorkExperiences?: CreatePreviousWorkExperiencesDto
+    inPrisonInterests?: CreateInPrisonInterestsDto
+    personalSkillsAndInterests?: CreatePersonalSkillsAndInterestsDto
+    futureWorkInterests?: CreateFutureWorkInterestsDto
+  }
+
+  export interface CreateWorkOnReleaseDto {
+    hopingToWork: 'YES' | 'NO' | 'NOT_SURE'
```

Comment:
All these "enums" that are written inline like this .... (I know I did the same in my PR) .... I'm wondering if there is now some duplication of them, and we can factor them out into proper enum types somehow?

You're `hopingToWork` one is definitely duplicated (I have that in my stuff that I did), and I wonder if any of the other "enums" here are now duplicated?

Reply:
Good point. I've switched to the enums that already existed, which Gary added a while back.